docs/config/index.md

```diff
@@ -659,7 +663,7 @@ If you are using coverage options with dot notation, don't forget to specify `--
 
 #### coverage.provider
 
-- **Type:** `'c8' | 'istanbul' | 'custom'`
+- **Type:** `'c8' | 'v8' | 'istanbul' | 'custom'`
 - **Default:** `'c8'`
```

Comment:
```suggestion
- **Type:** `'v8' | 'istanbul' | 'custom'`
- **Default:** `'v8'`
```

Do we see any blocker we could make `v8` the new default without breaking existing usage?

Reply:
Sounds good to me!

Reply:
There is now a message printed before prompting the installation of new package:

<img width="480px" src="https://github.com/vitest-dev/vitest/assets/14806298/d1401144-a684-4b86-aaa1-8d25c46a9f4c" />

This is only shown when `test.coverage.provider` or `--coverage.provider` option is not explicitly set.